scripts/setup.cfg

```
@@ -0,0 +1,36 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[metadata]
+name = rh-primaza-control
+version = 0.1.0
+author = Martin Mulholland
+author_email = mmulholl@redhat.com
```

For all: does it make sense to give this a more generic author & email?  For instance:
```suggestion
author = The Primaza Authors
author_email = primazactl@primaza.io
```

---

Fixed